src/main/java/io/netty/incubator/channel/uring/IOUringEventLoop.java

```diff
@@ -54,6 +57,9 @@
     private final byte[] inet4AddressArray = new byte[SockaddrIn.IPV4_ADDRESS_LENGTH];
     private final byte[] inet6AddressArray = new byte[SockaddrIn.IPV6_ADDRESS_LENGTH];
 
+    private final IOUringCompletionQueueCallback callback = IOUringEventLoop.this::handle;
+    private final Runnable submitIOTask = () -> getRingBuffer().ioUringSubmissionQueue().submit();
```

Comment:
Why don't this call back to `submitIO`? No need to handle completions?

Reply:
it now does the exact some as the `submitIO`